Automattic/wp-calypso
# Stats: Update Overview Components

This branch is part of the stats maintenance project and seeks to re-organize and cleanup the components that comprise the stats overview page.

__Files Moved__
- `stats-overview.jsx` -> `overview.jsx` - this is the main component for the overview page so leaving it at the root level in the stats directory
- `overview/` -> `stats-site-overview/` - this was always confusing name-wise.  This component builds out a single "card" of a site overview
- `module-site-overview-placeholder.jsx` -> `stats-overview-placeholder/index.jsx`

All files have been updated to ES6 syntax if applicable.

__To Test__
- Open up the [stats overview page](http://calypso.localhost:3000/stats) and verify everything looks the same as production
- Open up a stats insights page for a site, verify the "Today's Stats" section looks the same.
